### ISSUE TITLE
error C2039: “random_shuffle”: is not the member of "std"

### ISSUE THREAD

**Issue by Philip035** [ayushgargdroid/point-cloud-registration#1]
failed to compile the project

**Comment by ayushgargdroid** [comment]
Hi @Philip035.

Since this is a personal project still WIP, I haven't put up any dependencies or build instructions. Can I know how you are compiling the project?

For now, it doesn't have many dependencies, just PCL 1.8 and Eigen. Make sure you have at least CMake version 3.16 and C++ 17.

**Comment by Philip035** [comment]
Hi @ayushgargdroid 
  Thank you for the reply. I used Cmake(3.20) build the project with visual studio 15 2017 x64,and I changed C++14 to C++17, and pcl is 1.8.1
  I am really confused about the problem...

**Comment by ayushgargdroid** [comment]
Yeah, I don't think there should be problems building this package since it just requires standard packages. Are you on Linux?

Can you try cmake from the terminal?

```
cd {project_dir}
mkdir build
cmake ..
make
```

**Comment by Philip035** [comment]
sorry,I forget that case,I build it on windows
